solutions/cf-1006F.mdx

```diff
@@ -0,0 +1,128 @@
+---
+id: cf-1006F
+source: CF
+title: Xor Paths
+author: Chongtian Ma
+---
+
+[Offical Analysis (C++)](https://codeforces.com/blog/entry/60672)
+
+## Alternative Explanation
+This problem is actually pretty similar to the sample problem. As the official editorial states, we can manuever the grid in exactly $n + m - 2$ moves.
```

Comment:
what's the sample problem

Reply:
i meant the tutorial problem in the module what do you even call it

Reply:
"focus problem in the [x] module"